src/screens/LoginScreen/components/FormLogin/index.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+
+import VioletButton from '../../../../components/VioletButton';
+import WhiteButton from '../WhiteButton';
+import img from '../../assets/img/erroricon.png';
+import { sendGame } from '../../../../services/gameService';
+
+import styles from './index.module.scss';
+
+class FormLogin extends React.Component {
+  state = {
+    errorCode: false,
+    errorName: false,
+    errorServer: false,
+    code: '',
+    name: ''
+  };
+
+  handleSendGame = async data => {
+    const res = await sendGame(data);
+    if (res.status === 500) {
+      this.setState({ errorCode: true });
+    } else if (res.status === 404) {
+      this.setState({ errorServer: true });
+    } else {
+      console.log('redireccionamiento');
+    }
+  };
+
+  handleChange = event => {
+    const { value, name } = event.target;
+    this.setState({ [name]: value });
```

Comment:
Puedes poner todo en una sola línea:
`this.setState({ [name]: value, errorCode: false, errorName: false, errorServer: false  });`

Reply:
Gracias, hecho

Reply:
No lo veo resuelto 👀